src/lib.rs

```diff
@@ -60,3 +64,63 @@ impl OperatingSystem {
         }
     }
 }
+
+/// List of all supported operating systems.
+#[derive(clap::ValueEnum, Debug, Clone, PartialEq)]
+pub enum PackageManager {
+    Pacman,
+    Yay,
+    Apk,
+    AptGet,
+    Apt,
+    Api,
+    Pkg,
+    Dnf,
+}
+impl From<&OperatingSystem> for PackageManager {
+    /// Get the default package manager for the operating system.
+    fn from(os: &OperatingSystem) -> PackageManager {
+        match os {
+            OperatingSystem::Arch => PackageManager::Pacman,
+            OperatingSystem::Alpine => PackageManager::Apk,
+            OperatingSystem::Debian => PackageManager::AptGet,
+            OperatingSystem::Ubuntu => PackageManager::AptGet,
+            OperatingSystem::Fedora => PackageManager::Dnf,
+        }
+    }
+}
+
+/// Get the package manager to use.
+/// If the expected one is None, look for it in the environment variables.
+/// If it is not in the environment variables, guess it from the current operating system.
+///
```

Comment:
The environment variable name doesn't match with the one mentioned in the [README.md](https://github.com/Rignchen/Depot/blob/main/README.md) file which is `DEPOT_PM`